front/types/Video.ts

```diff
@@ -1,12 +1,12 @@
-type sizes = '144' | '240' | '480' | '720' | '1080';
+export type videoSize = '144' | '240' | '480' | '720' | '1080';
```

Comment:
should we enforce the possible sizes in the client or just type it as an integer and trust the server?

Reply:
I cannot type it as an integer, object keys are always strings no matter what.

I think it's better this way since it prevents possible mistakes. If we end up changing it often we can always revert to "anything goes" `string`